web/html/media/video-formats.md

```diff
@@ -52,15 +52,12 @@ AddType video/webm .webm
 
 Finish the sentence:
 
-In order for videos today to be supports on all major web browsers you should use either the `???` or `???` format.
-
-* .mp4
-* .webm
-* h.246
-* codec
-* .mp3
-* .oog
-* flash
+In order for videos to be supported on all major web browsers you should use the `???` format.
```

Comment:
Should we use **formats** instead of **format** now that the answers are in pairs?

>In order for videos to be supported on all major web browsers you should use the `???` formats.

Reply:
I would say so, yeah

Reply:
Actually in this grammatical context "format" seems correct